src/tests/user-prefs.cpp

```diff
@@ -69,7 +69,7 @@ TEST_F(rnp_tests, test_load_user_prefs)
 
         // find the key
         pgp_key_t *key = NULL;
-        assert_non_null(key = rnp_key_store_get_key_by_name(pubring, userid, NULL));
+        assert_non_null(key = rnp_tests_key_search(pubring, userid, NULL));
         assert_non_null(key);
```

Comment:
Once got here, we could also remove duplicate `assert_non_null(key)`, probably it was added by mistake during refactoring or other changes.